pyoverkiz/models.py

```diff
@@ -71,7 +71,7 @@ def __init__(
 @define(init=False, kw_only=True)
 class Location:
     creation_time: str
-    last_update_time: str
+    last_update_time: str | None = None
```

Comment:
Do we need to set it to None here as well? I think just below is enough.

Reply:
There are two different thing. Here we declare a property that’s an optional string, and below we declare a constructor taking as argument an optional string named `last_update_time`. They are only connected because in the constructor we assign the last one to the first one. 

By the way, you did the same for the `Setup` model :)

Reply:
I thought for newer models I only set it to None in the init params, not in the class variables anymore.

However I am happy to merge if this works. Reviewing from my phone.